indico/htdocs/js/indico/modules/surveys.js

```diff
@@ -91,7 +92,15 @@
         $('.js-add-question-dropdown').parent().dropdown({selector: '.js-add-question-dropdown'});
     }
 
-    global.setupSubmissionButtons = function setupSubmissionButtons() {
+    function setupSubmissionButtons() {
+        $('#select-all').on('click', function() {
+            $('#submission-list input').prop('checked', true).trigger('change');
```

Comment:
`input:checkbox` (or even add their name/class if they have one). While there's only one checkbox per entry right now, I prefer going the safe way in case someone ever decides to add something else there.